### PR TITLE
Fixed numeric discrepancies

### PR DESCRIPTION
**Pull request by tebayoso**
#### Description of the pull request (what is new / what has changed)

I did a review of the first page as a test to ensure external contributors can edit and add information to the documentation.

This change only aims to standardize the numeric values and remove slang.

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category(categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [x] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/welcome/welcome-to-elrond.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ title: Welcome to Elrond
 - **When:** Mainnet live, project started in 2018
 - **How:** Scales via Adaptive State Sharding, consensus via Secure Proof of Stake
 - **Highlights:**
-  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263k TPS in testnet)
+  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263,000 TPS in testnet)
```
Comment (SebastianMarian) [inline docs/welcome/welcome-to-elrond.md:12]: You can also change 15.000 -> 15,000 and 100k -> 100,000

Comment (tebayoso) [reply docs/welcome/welcome-to-elrond.md:12]:
```suggestion
  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, up to 100,000 (263,000 TPS in testnet)
```

File: docs/welcome/welcome-to-elrond.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ title: Welcome to Elrond
 - **When:** Mainnet live, project started in 2018
 - **How:** Scales via Adaptive State Sharding, consensus via Secure Proof of Stake
 - **Highlights:**
-  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263k TPS in testnet)
+  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, up to 100,000 (263,000 TPS in testnet)
```
Comment (tebayoso) [inline docs/welcome/welcome-to-elrond.md:12]:
```suggestion
  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100,000 (263,000 TPS in testnet)
```

File: docs/welcome/welcome-to-elrond.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ title: Welcome to Elrond
 - **When:** Mainnet live, project started in 2018
 - **How:** Scales via Adaptive State Sharding, consensus via Secure Proof of Stake
 - **Highlights:**
-  - Performance: 15.000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100k (263,000 TPS in testnet)
+  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, up to 100,000 (263,000 TPS in testnet)
```
Comment (SebastianMarian) [inline docs/welcome/welcome-to-elrond.md]: I think that "beyond" was the right term there, "up to" sounds like a limit which is not true

**Review by SebastianMarian — APPROVED** [review]

**Review by camilbancioiu — APPROVED** [review]

File: docs/welcome/welcome-to-elrond.md (hunk 3 above)
Comment (tebayoso) [reply docs/welcome/welcome-to-elrond.md]: I changed that to try the comment-commit comments, but apparently didn't work. Let me try again.

Comment (tebayoso) [inline docs/welcome/welcome-to-elrond.md]:
```suggestion
  - Performance: 15,000 current TPS, 6s block time, $0.001 tx cost, can scale beyond 100,000 (263,000 TPS in testnet)
```
